src/styles/blocks/utils/variables.scss

```diff
@@ -0,0 +1,5 @@
+$button-color: #00acdc;
+$text-color-1: #060b35;
+$text-color-2: #616070;
+$text-color-3: white;
+$colom-count: 1;
```

Comment:
It seems like there might be a typo in the variable name `$colom-count`. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), variable names should be written in kebab-case and be clear and meaningful. If you meant to use 'column count', the correct variable name should be `$column-count`.